app/workers/facilities/mental_health_reload_job.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'csv'
+require 'common/exceptions'
+require 'sentry_logging'
+
+module Facilities
+  class MentalHealthDownloadError < StandardError
+  end
+
+  class MentalHealthReloadJob
+    include Sidekiq::Worker
+    include SentryLogging
+
+    def fetch_mental_health_data
+      mental_file = Rails.root.join('lib', 'facilities', 'mental_health_data', 'mental_health_phone_numbers.csv')
+      CSV.read(mental_file, headers: true)
+    rescue => e
+      raise MentalHealthDownloadError, "Failed to download mental health data: #{e.cause}"
+    end
+
+    def update_cache(model, records)
+      records.each do |r|
+        ext = if r['Extension'] == '0' || r['Extension'] == 'NULL'
```

Comment:
Extracting this into a private(?) method would let you name it for better understanding -- I don't know a ton of context here but assume its something like `filter_records_with_bad_extensions` -- and would address the code climate issue as well. 

Without checking the original PR, maybe you could reject all of the bad ones in one method and then loop over the remaining ones unless you already combined that due to other suggestions 😄